hack/release.sh

```diff
@@ -14,12 +14,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -euo pipefail
+
 DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 EXAMPLES_DIR=${DIR}/../examples
 INTEGRATION_EXAMPLES_DIR=${DIR}/../integration/examples
 
+install_release_notes_helper() {
+  release_notes_workdir="$(mktemp -d)"
+  trap 'rm -rf -- ${release_notes_workdir}' RETURN
+
+  # See https://stackoverflow.com/questions/56842385/using-go-get-to-download-binaries-without-adding-them-to-go-mod for this workaround
```

Comment:
I would just add the tool to go mods!

Reply:
Can you explain a little how to do that? If I just add it with `go get .../release-notes`, then it will be lost on the next `go mod tidy`. Besides, I don't think this would install the tool in `$GOPATH/bin` or so.

Reply:
@corneliusweig One way to do it is to create a top level `tools` folder that is a sub-package, with it's own `go.mod`. In that folder, you add a `tools.go` file that imports `xxx/release-notes`. You can then run `go mod vendor` in that folder.
In the makefile, you'd run `cd tools; go install -mod=vendor xxx/release-notes`

Reply:
Here's a sample with `golangci-lint`

Reply:
@dgageot oh.. that's a nice way. I tried to stay away from vendor mode though, it seems to work ok either way.

Reply:
Why did you avoid vendoring? The advantage is that there's nothing to download on the Travis. It's a bit quicker and more resilient.

Reply:
It was adding 11MB of vendored code. Besides, it is only run on developer machines, right? So will not affect Travis anyway.

I know that you are not a big fan of go modules, so if you have a strong opinion in this case, I won't argue any longer :)

Reply:
Not a big fan indeed. In that case, it's actually very convenient. You are right that it doesn't run on Travis so, yes, vendoring is not so useful. Good point!